samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnContainerRunner.java

```diff
@@ -190,13 +197,25 @@ private void startContainer(Path packagePath,
       throw new SamzaContainerLaunchException("IO Exception when writing credentials to output buffer");
     }
 
+    Map<String, LocalResource> localResourceMap = new HashMap<>();
+    localResourceMap.put("__package", packageResource);
+
+    // include the resources from the universal resource configurations
+    try {
+      LocalizerResourceMapper resourceMapper = new LocalizerResourceMapper(config, yarnConfiguration);
```

Comment:
Currently, it calls the one from LocalizerResourceMapper and returns the localResourceMap for the context. 
There might be some legacy for package due to historical reason, and we plan to remove it later (TODO: SAMZA-1144 in the code), so I chose not to touch them now.